src/main/webapp/data/userMapData.json

```diff
@@ -162,6 +163,7 @@
       "Cegep de Bois-de-Boulogne",
       "Chetwynd Secondary School",
       "Collège Ahuntsic",
+      "College de Bois-de-Boulogne",
```

Comment:
Yes,  it appears to be. I'll remove the other entry as Collège de Bois-de-Boulogne, seems to be the standard name for it.